### DISCUSSION TITLE
Share autoortho between XP11 and XP12

### DISCUSSION THREAD

**Discussion by evwltrs** [kubilus1/autoortho#52] [Ideas]
Is this a feature that could be possible to implement? It would be super useful to not have to duplicate the autootho folders for each sim. Thanks

**Comment by ivotaHR** [comment]
I tested it worked. I made all folder junction with this tool 
https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html

Once , there was a problem with folder  z_autoortho\textures. delete folder solved the problem. (don't delete folder  z_autoortho\ **_textures** )
